.travis.yml

```diff
@@ -7,12 +7,12 @@ sudo: false
 
 before_install:
   - go version | (grep -q 'go1.[56]' || exit 0 && go get -u github.com/golang/lint/golint )
-  - go get github.com/vbatts/git-validation
+  - go get -u github.com/vbatts/git-validation
 
 install: true
 
 script:
   - go vet -x ./...
   - make .golint
-  - $HOME/gopath/bin/git-validation -run DCO,short-subject -v -range ${TRAVIS_COMMIT_RANGE}
+  - git-validation -run DCO,short-subject -v
```

Comment:
Maybe want `dangling-whitespace` too (from vbatts/git-validation#7), although that somewhat of a drive-by with this PR.  Still, it would be nice to avoid things like #380.